packages/elements/src/components/ino-avatar/ino-avatar.tsx

```diff
@@ -0,0 +1,109 @@
+import { Component, ComponentInterface, Prop, h, Element } from '@stencil/core';
+import classNames from 'classnames';
+import { hasSlotContent } from '../../util/component-utils';
+import primary from '../../assets/ino-avatar/primary.svg';
+import secondary from '../../assets/ino-avatar/secondary.svg';
+import primary_dashed from '../../assets/ino-avatar/primary_dashed.svg';
+import secondary_dashed from '../../assets/ino-avatar/secondary_dashed.svg';
+
+/**
+ * @slot icon-slot - For the icon to be displayed in the avatar.
+ */
+@Component({
+  tag: 'ino-avatar',
+  styleUrl: 'ino-avatar.scss',
+  shadow: false,
+})
+export class Avatar implements ComponentInterface {
+  @Element() el: HTMLElement;
+
+  /**
+   * The alternative text of the avatar image.
+   */
+  @Prop() alt?: string = '';
+
+  /**
+   * The initials of the avatar. Will be shown if no `src` is defined or `img` fails to load.
+   */
+  @Prop() initials: string = '';
+
+  /**
+   * The img of the avatar.
+   */
+  @Prop() src: string = '';
+
+  /**
+   * Flag to enable hover effect.
+   */
```

Comment:
That's a technical description of what it's doing on the inside. Maybe something along the lines "Makes the avatar indicate some kind of interactivity by adding hover effects and a pointing cursor" would be a better fit so the user knows when he should use this flag